torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6738,13 +6738,47 @@ def sample_inputs_tril_triu(op_info, device, dtype, requires_grad, **kwargs):
     make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
     cases = (((M, M), ()),
              ((M, M), (2,),),
-             ((S, M, M), ()),
-             ((S, M, M), (2,)),
+             ((M, S), ()),
+             ((M, S), (-1,)),
+             ((M, M), (2,),),
+             ((S, M, S), ()),
+             ((S, M, S), (2,)),
              ((3, 3, S, S), ()),)
 
     for shape, args in cases:
         yield SampleInput(make_arg(shape), args=args)
 
+def sample_inputs_trilu_indices(op_info, device, dtype, requires_grad, **kwargs):
+    # (row, col, offset)
+    args_list = ((1, 1),
+                 (3, 3, 1),
```

Comment:
I don't think you need all these inputs as sample inputs (a lot of tests are run on sample inputs), it's better to have "interesting" small inputs as sample, and everything else as reference

Reply:
Reasonable, cut those down.

On a related note, prior to me adding the OpInfos for `triu_indices/tril_indices` they were already being tested by some tests in `test/test_cuda.py` and `test/test_tensor_creation_ops.py`, with some shared code in `common_methods_invocations.py`. I haven't looked at those very closely, but they are probably redundant with having the OpInfo based tests - do you think we should consider dropping them?
Since we are going to need OpInfos for all the tensor creation ops for PrimTorch, maybe we will want to drop `test/test_tensor_creation_ops.py` entirely at some point...

Otherwise, I made all the changes you suggested, let me know if there's anything else.

Reply:
Yes, removing tests in test_tensor_creation_ops and test_cuda sounds great! (Doesn't need to be in this PR).

Reply:
Ok, will do it in a separate PR.